Cartfile.resolved

```diff
@@ -1 +1,2 @@
-github "krzyzanowskim/CryptoSwift" "0.2.2"
+github "krzyzanowskim/CryptoSwift" "0.2.3"
+github "jspahrsummers/xcconfigs" "0.9"
```

Comment:
To remove boilerplate build settings in xcodeproj. The build targets are now based on some `.xcconfig` files.